src/views/Patient/MedReqDropDown/MedReqDropDown.tsx

```diff
@@ -318,6 +321,11 @@ function MedReqDropDown(props: MedReqDropDownProps) {
                         </Button>
                       </Grid>
                     )}
+                    <Grid item>
+                      <Button variant="contained" onClick={sendOrderSelect}>
```

Comment:
What's the use case for this button?  If we just want to be able to resend the same hook, we should use `SubmitToRems` to avoid redoing the hydration step.  Otherwise this is fine.

Reply:
Ah, thank you so much for catching that. The button's use case is to send a `rems-order-select` Network request. I'm not yet very familiar with how/what hydration does- is it an expensive/redundant operation?

Reply:
it's not particularly expensive but no reason to redo it if we don't have to.  It depends on the use case, if we need to remake the hook or if we just want to re-try the sending of the current hook.

Reply:
I see. I misunderstood what "use case" meant here. The ticket specifies just resending, not remaking the hook. 

I assume (as my educated guess) remaking the hook depends on some behind-the-scenes stuff like generating new SMART links, or has to do with asynchronous stuff like tokens expiring? Although I know the token expires after a certain amount of time. Or just the list of medications changing on the fly, like in real-time (but that's not happening in our app). 

Sorry for the rambling, @plarocque4 might be able to answer this better than I can- what might be the significance of remaking the CDS Hook?

Reply:
I don't think we need to bother doing the prefetch again. This is just a convenience feature to send the hook again without having to select a different med then the same one again. It's mostly for demonstration purposes, but basically does a check to see what forms are still needed.